openslides/motions/models.py

```diff
@@ -697,6 +698,57 @@ def get_root_rest_element(self):
         return self.motion
 
 
+class MotionChangeRecommendation(RESTModelMixin, models.Model):
+    """
+    A MotionChangeRecommendation object saves change recommendations for a specific MotionVersion
+    """
+
+    access_permissions = MotionChangeRecommendationAccessPermissions()
+
+    motion_version = models.ForeignKey(
+        MotionVersion,
+        on_delete=models.CASCADE,
+        related_name='change_recommendations')
+    """The motion version to which the change recommendation belongs."""
+
+    type = models.PositiveIntegerField()
+    """ Insertion (1), Deletion (2) or Replacement (0) """
```

Comment:
Docstring should be consistent. Sometimes they start with whitespace, sometimes not.